test/jasmine/tests/tickformatstops_test.js

```diff
@@ -0,0 +1,307 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var Axes = require('@src/plots/cartesian/axes');
+var Fx = require('@src/components/fx');
+var d3 = require('d3');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var selectButton = require('../assets/modebar_button');
+var fail = require('../assets/fail_test');
+
+var mock = require('@mocks/tickformatstops.json');
+
+function getZoomInButton(gd) {
```

Comment:
Can we merge this file into `axes_test.js`? I don't see the need to break src file / test file symmetry here.

Reply:
done in https://github.com/plotly/plotly.js/pull/1965/commits/0e6747fef8825e716936fcf6c85e592b54aebcbb